myst_nbs/variational_inference/pathfinder.myst.md

```diff
@@ -0,0 +1,82 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: pymc4
+  language: python
+  name: pymc4
+---
+
+(pathfinder)=
+
+# Pathfinder Variational Inference
+
+:::{post} Sept 30, 2022 
+:tags: variational inference, jax 
+:category: advanced 
+:author: Thomas Wiecki
+:::
+
++++
+
+Pathfinder is a variational inference algorithm that produces samples from the posterior of a Bayesian model. It compares favorably to the widely used ADVI algorithm. On large problems, it should scale better than most MCMC algorithms, including dynamic HMC (i.e. NUTS), at the cost of a more biased estimate of the posterior. For details on the algorithm, see the [arxiv preprint](https://arxiv.org/abs/2108.03782).
```

Comment:
I would add the reference to pathfinder paper as citation instead of only url. You'll need to add the bibtex citation to https://github.com/pymc-devs/pymc-examples/blob/main/examples/references.bib and then use the syntax described in https://docs.pymc.io/en/latest/contributing/jupyter_style.html#references.